### DISCUSSION TITLE
Tilting Grid Size

### DISCUSSION THREAD

**Discussion by JEBurroughs** [mriscoc/Ender3V2S1#337] [Q&A] [answered]
I noticed that no matter what you change Advanced/ Mesh Leveling/ Tilting Grid Size to it and save, it always reverts back to 1. Just curious why?

**Comment by mriscoc** [comment, marked as answer]
You are right, currently Tilting grid size is not a saved parameter because it is considered only a correction parameter as flow and speed.

**Reply by JEBurroughs** [reply]
Cool, now I understand it much better.
